db/crud.go

```diff
@@ -2332,14 +2421,18 @@ func (col *DatabaseCollectionWithUser) getChannelsAndAccess(ctx context.Context,
 			expiry = output.Expiry
 			err = output.Rejection
 			if err != nil {
-				base.InfofCtx(ctx, base.KeyAll, "Sync fn rejected doc %q / %q --> %s", base.UD(doc.ID), base.UD(doc.NewestRev), err)
-				base.DebugfCtx(ctx, base.KeyAll, "    rejected doc %q / %q : new=%+v  old=%s", base.UD(doc.ID), base.UD(doc.NewestRev), base.UD(body), base.UD(oldJson))
-				col.dbStats().Security().NumDocsRejected.Add(1)
-				col.collectionStats.SyncFunctionRejectCount.Add(1)
-				if isAccessError(err) {
-					col.dbStats().Security().NumAccessErrors.Add(1)
-					col.collectionStats.SyncFunctionRejectAccessCount.Add(1)
+				dryRunMsg := ""
+				if !dryRun {
+					dryRunMsg = "[DRY RUN]"
+					col.dbStats().Security().NumDocsRejected.Add(1)
+					col.collectionStats.SyncFunctionRejectCount.Add(1)
+					if isAccessError(err) {
+						col.dbStats().Security().NumAccessErrors.Add(1)
+						col.collectionStats.SyncFunctionRejectAccessCount.Add(1)
+					}
 				}
+				base.InfofCtx(ctx, base.KeyAll, "%s Sync fn rejected doc %q / %q --> %s", dryRunMsg, base.UD(doc.ID), base.UD(doc.NewestRev), err)
```

Comment:
```suggestion
				base.InfofCtx(ctx, base.KeyAll, "%sSync fn rejected doc %q / %q --> %s", dryRunMsg, base.UD(doc.ID), base.UD(doc.NewestRev), err)
```
(to avoid whitespace logging change in the non-dry-run case)